### ISSUE TITLE
Could you release your method for creating synthetic line drawings of 3D shapes

### ISSUE THREAD

**Issue by xiaomingjie** [happylun/SketchModeling#6]
HI @happylun 

Thank you for your amazing work!
I would like to use more categories of 3D shapes to get through this but I can't find the code to create line drawings.
Looking forward to your reply.

**Comment by happylun** [comment]
Most of our synthetic sketches are generated using code from the Suggestive Contours project (http://gfx.cs.princeton.edu/proj/sugcon/). By tuning the parameters in their code a bit you can get different variations of sketches.

**Comment by xiaomingjie** [comment]
Thanks a lot!

**Comment by LiuZhenyan-Wuzhong** [comment]
> Most of our synthetic sketches are generated using code from the Suggestive Contours project (http://gfx.cs.princeton.edu/proj/sugcon/). By tuning the parameters in their code a bit you can get different variations of sketches.

Excuse me, how could you render a batch of the sketch of line drawing of special views and output them?  The exe file in the project you have cited can not be operated by keyboard input and there is nothing except the input model. Have you changed the C++ source code? 
Thanks for your attention.

**Comment by happylun** [comment]
Yes I modified the source code to run it with custom viewpoint config. It's OpenGL code so should be easy to specify your own viewpoint config in the code.
